app/main.py

```diff
@@ -1,13 +1,23 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    } \
+        if len(error) > 0 else {}
```

Comment:
```suggestion
    }
```

Since we are not expecting that someone will call this function with wrong data. Otherwise, you need to check if all keys exist in this dict, but this is not needed to be implemented in scope of this task.